docs/authentication/fine-grained-auth-token.md

```diff
@@ -0,0 +1,35 @@
+# Authentication with Fine-grained PAT (optional)
+
+For repositories that want to detect their classic Branch Protection rules, or webhooks, we suggest you create a fine-grained Personal Access Token (PAT) that Scorecard may use for authentication.
```

Comment:
does the default `GITHUB_TOKEN` work for webhook too?
Does the sentence `Scorecard can run successfully with the workflow's default `GITHUB_TOKEN`, which is our recommended approach.` not apply to webhooks then?

Reply:
no, it doesn't work for webhook and will always need an admin read token.
But webhook is experimental and not accessible from Scorecard Action anyway